tools/autobahn

```diff
@@ -104,6 +106,34 @@ def expand_refvar(module):
     return module
 
 
+def expand_overrides(overrides):
+    for key in overrides:
+        overrides[key] = walk(overrides[key], expand_slot_shorthand)
+        overrides[key] = walk(overrides[key], expand_shortdef)
+    return overrides
+
+
+def process_custom_overrides(tree):
```

Comment:
It would be better to pass `overrides` to this function, I think.

Reply:
overrides is globally accessible, but I think is fine either way.